<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_listen">
      <property name="text">
       <string>开始侦听</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="le_command"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_Create"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设备(&amp;D)</string>
    </property>
    <addaction name="action_link"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Create"/>
   <addaction name="action_Save"/>
   <addaction name="action_SaveAs"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Exit"/>
  </widget>
  <action name="action_Create">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/E:/Download/Icon/file-add.png</normaloff>:/myImages/E:/Download/Icon/file-add.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/E:/Download/Icon/folder-open.png</normaloff>:/myImages/E:/Download/Icon/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/E:/Download/Icon/folder (1).png</normaloff>:/myImages/E:/Download/Icon/folder (1).png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/E:/Download/Icon/fund.png</normaloff>:/myImages/E:/Download/Icon/fund.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/E:/Download/Icon/pause-circle (1).png</normaloff>:/myImages/E:/Download/Icon/pause-circle (1).png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_link">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/E:/Download/Icon/fund.png</normaloff>:/myImages/E:/Download/Icon/fund.png</iconset>
   </property>
   <property name="text">
    <string>连接(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myResources.qrc"/>
 </resources>
 <connections/>
</ui>
